javascript/javascript2/week1/data.json

```
@@ -2,307 +2,401 @@
   {
     "id": 1,
     "title": "Flaming Hot Nachos",
-    "imageUrl": "https://hips.hearstapps.com/hmg-prod/images/flaming-hot-nachos-vertical-6570e53a67667.jpg?crop=0.8332502078137989xw:1xh;center,top&resize=980:*.jpg",
+    "picture_url": "https://hips.hearstapps.com/hmg-prod/images/flaming-hot-nachos-vertical-6570e53a67667.jpg?crop=0.8332502078137989xw:1xh;center,top&resize=980:*.jpg",
+    "preparation_time": "20 mins",
+    "total_time": "40 mins",
+    "yield": "4 servings",
     "ingredients": [
-      "½ cup sour cream",
-      "500 gr ground beef",
-      "2 cups crushed Flamin' Hot Cheetos",
-      "20 gr tortilla chips",
-      "150 gr shredded Mexican cheese",
-      "1 can black olives, drained",
-      "1 can black beans, drained",
-      "1 red onion, chopped",
-      "2 cloves garlic, chopped",
-      "¼ tsp cumin",
-      "Salt & pepper"
+      { "NAME": "½ cup sour cream" },
```

I would use a lowercase attribute to be consistent with the rest of the file.

```suggestion
      { "name": "½ cup sour cream" },
```